### DISCUSSION TITLE
萬事起頭難

### DISCUSSION THREAD

**Discussion by TW-Maverick** [1101-GS4719-C/Python-Community-GS4719-C#1404] [General]
很常有想法，但都難以付諸行動，像古人說的「天下無難事，只怕有心人」，事情開始的行動是最難的。
我覺得會有拖延症，有很大的一部份是因為做一件事之前要先規劃？導致不想現在就去面對，直到不得已不面對。

不知大家的想法是什麼？

**Comment by a3240281370** [comment]
我也是很會拖延的人哎，我的方法就是一知道有一件事要做的當下就開始先做一部份，比如說功課派下來的當天我就先寫一部份，至少先把最困難的起頭解決了，真的能大幅減緩拖延症！
ps:不用一開始就想著一次解決否則可能造成反效果

**Comment by ziihui** [comment]
做事之前的規劃確實很重要，但也容易讓人陷入無限的思考與猶豫中。建議可以先制定一個簡單的行動計畫，將任務分成小步驟逐一完成，並設定明確的時間限制。此外，鼓勵自己先行動起來，遇到問題再及時調整計畫，逐漸建立自信與行動力。

**Comment by poweichen00** [comment]
我也常很有想法，但要起頭時就會退縮，然後時間就一點一滴的過去了，我可以試試樓上的方法，先做一部分至少有個開始。

**Comment by shuxian12** [comment]
我覺得萬事起頭非常難，但是要堅持下去更難，因為通常你想做一件事，如果當下有空的話，就不需要思考直接去做就好，但可能到了之後事情越來越多，或是有天不小心中斷了這件事，那可能就再也撿不回來了！！

**Comment by Jamesmoneya** [comment]
我有想過要健身，想了一陣子，遲遲沒有那個心力真的跑去健身房健身，只是起初有一個想要的想法，真的要做下去真的好難希望我暑假能真的去健身

**Comment by aa103967622** [comment]
我覺得要有那種行動力去開始計畫取決於自己對這件事情的企圖心，你是否渴望見到達成目標後的自己，或是你等不及在過程中獲得成長了！但有時候計畫好的事情一直沒有開始是因為你的想法還停留在計畫階段，只是腦中想好所有該做的事情還有行程表，但是卻沒有進階到開始計畫的階段，空有完美計畫，卻毫無開始的跡象，或許大家該利用完成之後的獎勵去激發自己的動力，比如，開始健身後，我的運動表現會得到提升，或是體態變得精壯好看，請這樣去鼓勵自己邁出第一步吧！

**Comment by alicia0725** [comment]
我也是一個超級容易拖延的人，尤其是晚上比較有想法我的想法都會是喔我明天要做什麼什麼，結果第二天一忙起來又會放棄，我自己現在的方法會是前一天列一個清單，隔天會比較有動力把他全部做完

**Comment by 1104030360** [comment]
以前的我也是非常靠嘴上工夫，只會講不會做，後來我認知到只要撐過開頭那段的不舒服感覺，後面做這件事就會順暢許多，因此之後我都會謹記要把開頭不舒服感覺當成功的養分，這樣開始任何計畫都會容易許多

**Comment by zephyrxin01** [comment]
我覺得如果有學伴或是朋友能相互督促可以幫助自己擺脫拖延症！我之前一直想要練習英文口說，但發現自己一個人要練習很不容易，不只常常pass掉規劃好要練習的時間，也無法得到別人的反饋。後來有了朋友一起練習之後，有了更多樂趣也更不會有理由拖延。

**Comment by ALAN30359704** [comment]
我認為萬事起頭難，對我而言並不算是一件很難的事情，反而是要如何持之以恆來完成這一件事情對我而言我覺得是比較難的，因為常常我常常是三分鐘熱度的想要去完成一件事，但是沒有去想到背後所需要付出的代價是時間的所需要消耗的精力。

**Comment by bestbanana295** [comment]
真的，有時候覺得有鞋事情很酷很想去試試看，但是還沒開始熱情就熄滅了，所以已開始真的是最難的，要努力撐過去！所以我覺得藝閣武林不乳很多人的武林，要很多朋友才可以一起撐過去。

**Comment by alstinlin** [comment]
對我而言如果在開始之前有設立一個目標的話，會比較容易堅持下去，當然也要是一個可以達成的目標，因為達成後就會有動力繼續往前進，除此之外我也蠻認同對於比較困難的挑戰，如果身邊有朋友一起努力的話就能夠堅持得比較久~

**Comment by youtinglin** [comment]
我覺得會萬事起頭難並不是自己能力差，而是缺少動力，或是對於一開始就有太高的期望，過程總是辛苦的，因此可以和別人訴苦，藉由朋友間的支持和鼓勵，讓事情逐漸能上軌道，先設立短期目標並達成，當多個短期目標完成，也就達到自己的最終目標，不要太好高鶩遠，一步步慢慢地完成就行啦。

**Comment by Elaine2899** [comment]
我也認同"萬事起頭難"，但我的個人經驗倒不是因為缺少動力，而是缺少自信。最近想開始接家教，但缺少相信自己可以的信心，也擔心誤人子弟，更擔心小孩不喜歡我等等。但客觀地來說，我應該不需要擔心這麼多，只需要好好備課應該就已經勝過很多家教了，但不知為何我就是沒有信心可以做到我心目中的標準，而導致我不敢投遞履歷，最後去找家教補習班，希望可以從薪水較低的地方開始訓練自己。

**Comment by xiang305** [comment]
我認同萬事起頭難，所以我認為比起在做事前先進行良好的規劃，有些事情不如先做了再說。不要去害怕失敗，而是讓自己從失敗中學到經驗，我認為有些事情是要去親身經歷才能夠想通的，就像人生一定要走一些冤枉路一樣。總而言之，如果不付諸行動，只在腦袋裡空想，一切都不會改變的

**Comment by Evening218** [comment]
我常常會在有一堆待辦事項的時候拖延症發作,但是其實只要開始做事,就會很自然地投入進去,所以真的是開始比較難,我覺得一旦進入狀況反而會變得很專心而且效率也高
